### PR TITLE
Bump version and add platforms.

### PR DESCRIPTION
**Pull request by oursland**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: conda-forge.yml [hunk 1]
```diff
@@ -4,3 +4,8 @@ github:
   tooling_branch_name: main
 conda_build:
   pkg_format: '2'
+build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
+  osx_arm64: osx_64
+  win_64: win_64
```
Comment (h-vetinari) [inline conda-forge.yml]: You don't need this here for windows, but you do need to delete
https://github.com/conda-forge/backtrace-feedstock/blob/70f9775450e285ceb6d27683ad452abb13b8e992/recipe/meta.yaml#L15

**Comment by oursland** [comment]
Unfortunately, the Windows build did not succeed and it will require additional investigation to understand the appropriate approach to build an automake/autoconf project on a Windows platform.  Consequently, the Windows build was removed until a later date.

**Comment by bluescarni** [comment]
@oursland thanks! I will go ahead and merge this for the time being.

FYI, I am not sure about the details, but I think it is now possible to use a GCC-based Windows toolchain on conda-forge (including autotools). E.g., see the GMP recipe here:

https://github.com/conda-forge/gmp-feedstock/blob/main/recipe/meta.yaml

**Comment by h-vetinari** [comment]
It's also possible to use https://github.com/conda-forge/autotools_clang_conda-feedstock for using clang+autotools on windows.

**Pull request merged**